src/mesh/mesh.cxx

```diff
@@ -377,9 +392,26 @@ void Mesh::createDefaultRegions(){
   addRegion2D("RGN_NOY", Region<Ind2D>(0, LocalNx - 1, ystart, yend, 0, 0, LocalNy, 1,
                                        maxregionblocksize));
 
+  //Perp regions
+  addRegionPerp("RGN_ALL", Region<IndPerp>(0, LocalNx - 1, 0, 0, 0, LocalNz - 1, 1, LocalNz,
+                                       maxregionblocksize));
+  addRegionPerp("RGN_NOBNDRY", Region<IndPerp>(xstart, xend, 0, 0, 0, LocalNz - 1, 1, LocalNz,
+                                           maxregionblocksize));
+  addRegionPerp("RGN_NOX", Region<IndPerp>(xstart, xend, 0, 0, 0, LocalNz - 1, 1, LocalNz,
+                                       maxregionblocksize));
+  addRegionPerp("RGN_NOY", Region<IndPerp>(0, LocalNx - 1, 0, 0, 0, LocalNz - 1, 1, LocalNz,
```

Comment:
Does `NOY` make sense for `indPerp` as it only has a single y value?

Reply:
No, but I think this gives the expected behaviour.  IndPerp has no Y guard cells, so returning the region without guard cells is just itself.